assets/installer/Components/Final.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Button, Card, CardContent, CardHeader } from '@mui/material'
+
+export const Final = () => {
+
+  return (
+    <Card>
+      <CardHeader title="Final" />
+      <CardContent>
+        <div>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet assumenda at, atque aut beatae consequuntur
```

Comment:
Change message to `Installation completed successfully. Reload page, if you wish to configure services.`.